examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<p>Looks like a scipy issue in the multinomial. This seems to run without problems on my machine:</p><p> </p><p><code>p = sp.stats.dirichlet(conc * frac).rvs(size=n_samples)</code></p><p><code>y = np.stack([np.random.multinomial(20, p[i, :]) for i in range(n_samples)])</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Or even</p><pre class="ql-syntax" spellcheck="false">np.vstack([sp.stats.multinomial(n=total_count, p=p[i, :]).rvs() for i in range(n_samples)])
</pre><p>works for me</p>